build/webpack.config.dev.js

```diff
@@ -54,6 +57,9 @@ module.exports = (env) => {
         }
       ]
     },
+    output: {
+      publicPath: env.APP_URL.indexOf('dev') !== -1 ? '/dev/' : '/'
```

Comment:
would `env.BASE_URL` work? (here and in the `base:` place)

Reply:
That could also work, though we'd need to update the commands in `package.json` to set `env.BASE_URL` since those don't exist right now.

Reply:
ah. nevermind then.